pl_examples/basic_examples/lightning_module_template.py

```diff
@@ -19,7 +19,24 @@
 
 class LightningTemplateModel(LightningModule):
     """
-    Sample model to show how to define a template
+    Sample model to show how to define a template.
+
+    Example:
+
+        >>> # define simple Net for MNIST dataset
+        >>> params = dict(
+        ...     drop_prob=0.2,
+        ...     batch_size=2,
+        ...     in_features=28 * 28,
+        ...     learning_rate=0.001 * 8,
+        ...     optimizer_name='adam',
+        ...     data_root='./datasets',
+        ...     out_features=10,
+        ...     hidden_dim=1000,
+        ... )
+        >>> from argparse import Namespace
+        >>> hparams = Namespace(**params)
+        >>> model = LightningTemplateModel(hparams)
```

Comment:
you can just pass in a dict now... no need to cast as Namespace

Reply:
it is not there yet (master) I will prepare follow-up pr with this chnage